packages/mui-base/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/base",
-  "version": "5.0.0-beta.69",
+  "version": "5.0.0-beta.70",
```

Comment:
We are releasing Base UI from master, right?

Reply:
We're not releasing MUI Base unless there's a reason to do so, is there?

Reply:
I thought we stopped with this once we saw that it created problems with the master vs next branch. I will revert it, maybe I am wrong. cc @aarongarciah do you remember this? I think you faced an issue when releasing some time ago.

Reply:
The last issue I remember was when we wanted to support React 19 in Joy, which required Base. But IIRC, after and before that, we weren't releasing neither frequently.

Reply:
Yeah I think we landed on not releasing MUI Base unless it was necessary.

Reply:
Reverted this bump.